scripts/test/runtests.sh

```diff
@@ -6,6 +6,12 @@
 
 set -e
 
+#force tests to use stage 2 output
+export StartPath=$PATH
```

Comment:
This file should be agnostic to which version of the binaries it is testing. 

It will be used to test binaries in packages and other places than the stage2 directory. 
This logic should be in the calling script.